### DISCUSSION TITLE
微调时，apex相关报错

### DISCUSSION THREAD

**Discussion by FZGgithub** [THUDM/CogVLM#347] [Q&A]
我在4张A6000上微调cogvlm，使用的模型是hf的，demo可以成功多卡推理，但是微调时会报错：from pyramid.session import UnencryptedCookieSessionFactoryConfig。我的服务器是kylin的，参照了网上关于apex的安装方法，未能成功。
请问，这个该如何处理呢

**Comment by zxk1234** [comment]
我是这么安装的：
! git clone https://github.com/NVIDIA/apex.git
% cd apex
! python setup.py install --cuda_ext --cpp_ext
! pip install -v --disable-pip-version-check --no-cache-dir --no-build-isolation --config-settings "--build-option=--cpp_ext" --config-settings "--build-option=--cuda_ext" ./
%cd ..
